igmspec/ingest/myers.py

```diff
@@ -4,13 +4,143 @@
 
 import numpy as np
 import os
-import pdb
-
+#import pdb
+from xastropy.xutils import xdebug as xdb
 from astropy.table import Table
 from astropy.io import fits
+from astropy import units as u
+
+# What should this matching tolerance be?? Set to 2.0" for now
+def add_to_hdf(hdf, Z_MIN = 0.1, Z_MAX = 7.1, MATCH_TOL = 2.0*u.arcsec):
+
+    from astropy import units as u
+    from astropy.table import QTable, Table, Column, hstack, vstack
+    from astropy.coordinates import SkyCoord, match_coordinates_sky, search_around_sky
+
+    ## SDSS/BOSS data stuff
+    specfile = os.getenv('RAW_IGMSPEC') + '/sdss/specObj-dr12_trim.fits'
+    spec = Table.read(specfile)
+    # Read in select columns from DR12 photometry. This and the file above are aligned
+    posfile = os.getenv('RAW_IGMSPEC') + '/sdss/photoPosPlate-dr12_trim.fits'
+    phot = Table.read(posfile)
+
+    # Trim to QSO, Specprimary, spec2d called it a QSO, redshift flag cuts, sanity check on coords
+    itrim = (spec['SPECPRIMARY'] == 1) & \
+            [('QSO' in q) for q in spec['CLASS']] & \
+            (spec['ZWARNING'] < 5) & \
+            (spec['PLUG_RA'] >= 0.0) & (spec['PLUG_RA'] <= 360.0) & \
+            (np.abs(spec['PLUG_DEC']) <= 90.0)
+    spec = spec[itrim]
+    phot = phot[itrim]
+    sdss_boss1 = hstack([spec, phot], join_type='exact')
+    # Add SDSS prefix to all SDSS tags
+    for key in sdss_boss1.keys():
```

Comment:
So in practice this is very easy to do with the spectroscopic MJD of the observation.  The following is the easiest way to distinguish. In other words, all data with MJD < 55049 was SDSS. 

sdss = spec_mjd  < 55049
boss = ~sdss